Update bibliography.bib

YihanLiu4023 opened the pull request.

YihanLiu4023 committed:
Update bibliography.bib

YihanLiu4023 committed:
Update bibliography.bib

YihanLiu4023 commented:
@TuomasBorman Hi Tuomas, I just created a new PR to see if this is correct and no conflicts. Please let me know if this workds! Thank you!

YihanLiu4023 committed:
Update MMUPHin_meta_analysis.qmd

YihanLiu4023 committed:
Update MMUPHin_meta_analysis.qmd

YihanLiu4023 committed:
Update MSEA.qmd

YihanLiu4023 committed:
Update MSEA.qmd

YihanLiu4023 committed:
Update IntegratedLearner.qmd

YihanLiu4023 committed:
Update IntegratedLearner.qmd

antagomir commented:
Looks good. I will wait for @TuomasBorman to merge.

TuomasBorman commented on inst/assets/bibliography.bib:
```diff
@@ -3,18 +3,6 @@
 
 
 
-@Article{Willis2019,
-  author =	 {Amy D. Willis},
-  title =	 {Rarefaction, Alpha Diversity, and Statistics},
-  journal =	 {Frontiers in Microbiology},
-  year =	 {2019},
-  volume =	 10,
-  doi =		 {10.3389/fmicb.2019.02407}
-}
-
-
```
Why this is removed?

TuomasBorman commented on inst/assets/bibliography.bib:
```diff
@@ -1941,235 +1969,217 @@ @article{Haidich2010
+    doi = {https://doi.org/10.1038/s41587-023-01872-y},
     year = {2023}
 }
-
-@article{Legendre2001,
-  author = {Legendre, Pierre and Gallagher, Eugene D.},
-  title = {Ecologically Meaningful Transformations for Ordination of Species Data},
-  journal = {Oecologia},
-  month = oct,
-  volume = {129},
-  number = {2},
-  pages = {271--280},
-  doi = {10.1007/s004420100716},
-  year = {2001}
-}
-
-@article{Schloss2024rarefaction1,
-    author = {Patrick D. Schloss},
-    title = {Rarefaction is currently the best approach to control for uneven sequencing effort in amplicon sequence analyses},
-    journal = {{mSphere}},
-    volume = {9},
-    number = {2},
-    pages = {e00354-23},
-    doi = {10.1128/msphere.00354-23},
-    year = {2024}
-}
-
-@article{Schloss2024rarefaction2,
-    author = {Patrick D. Schloss},
-    title = {Waste not, want not: revisiting the analysis that called into question the practice of rarefaction},
```
Why these are removed?

antagomir replied:
I would keep the refs that are there. We might like to cite them later when finalizing the text.

TuomasBorman reviewed (changes requested):
Looks good!

There are couple of references that are completely removed. That is ok, if they are not used in book, just wanted to check

antagomir committed:
Merge branch 'devel' into Reference-Fix

TuomasBorman committed:
up

TuomasBorman committed:
up

TuomasBorman committed:
up

Merge branch 'Reference-Fix' of github.com:YihanLiu4023/OMA into Reference-Fix

# Conflicts:
#	inst/assets/bibliography.bib

TuomasBorman committed:
Merge branch 'devel' into Reference-Fix
